packages/docs/modules/vuestic.ts

```diff
@@ -25,8 +25,10 @@ export default defineNuxtModule<VuesticOptions>({
     nuxt.options.alias['@vuestic/ag-grid-theme'] = resolve(__dirname, '../../ag-grid-theme/src/styles/index.scss');
     nuxt.options.alias['vuestic-ui/styles/typography.css'] = resolve(__dirname, '../../ui/src/styles/typography/typography.scss');
     nuxt.options.alias['vuestic-ui/styles/grid'] = resolve(__dirname, '../../ui/src/styles/grid/grid.scss');
+    nuxt.options.alias['vuestic-ui/styles/grid.css'] = resolve(__dirname, '../../ui/src/styles/grid/grid.scss');
```

Comment:
Can't understand, why we need these two aliases.

Reply:
Same. But it doesn't look harmful.

Reply:
Otherwise docs build fails.